### PR TITLE
Added 'Options' option for Directory-sections (bsc#707836)

### PR DESCRIPTION
**Pull request by cwh42**

### CODE REVIEW

File: src/modules/YaPI/HTTPDModules.pm [hunk 1]
```diff
@@ -118,7 +118,8 @@ textdomain "http-server";
                     position  => 70,
                     directives=> [ { option => "ScriptLog", 		"context" => [ "Server", "Virtual" ] },
 				   { option => "ScriptLogBuffer", 	"context" => [ "Server", "Virtual" ] },
-				   { option => "ScriptLogLength", 	"context" => [ "Server", "Virtual" ] }
+				   { option => "ScriptLogLength", 	"context" => [ "Server", "Virtual" ] },
+                                   { option => "Options",       	"context" => [ "Directory" ] }
```
Comment (cwh42) [inline src/modules/YaPI/HTTPDModules.pm]: tab vs spaces - but ... really?

Comment (cwh42) [inline src/modules/YaPI/HTTPDModules.pm]: Ok, ok - fixed it

File: package/yast2-http-server.changes [hunk 2]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Tue Oct 25 15:16:37 UTC 2016 - cwh@suse.com
+
+- Added 'Options' option for Directory-sections (bsc#707836)
+- 3.1.8
```
Comment (ancorgs) [inline package/yast2-http-server.changes]: We agreed to jump to 3.2.0 in master and we should be consistent with that, even if it's quite unlikely we need to fix yast2-http-server for SLE12 or CASP.

**Comment by ancorgs** [comment]
LGTM

**Pull request merged**